plugins/modules/site_intent.py

```diff
@@ -934,75 +941,78 @@ def get_diff_deleted(self, config):
             self.delete_single_site(item['id'], item['name'])
 
         # Delete the final parent site
-        self.delete_single_site(site_id, self.want.get("site_name"))
-        msg = "Site - {0} and it's child sites deleted successfully".format(self.want.get("site_name"))
-        self.result['response'] = msg
-        self.log(msg)
+        self.delete_single_site(site_id, site_name)
+        self.msg = "Site - {0} and it's child sites deleted successfully".format(site_name)
```

Comment:
self.message = "The site {0} and its child sites have been deleted successfully".format(site_name)

Reply:
added.